src/petals/server/handler.py

```diff
@@ -528,14 +531,18 @@ def _check_uids(self, uids: str) -> Tuple[ModuleUID, ...]:
 
     @contextlib.asynccontextmanager
     async def _allocate_cache(
-        self, backends: Sequence[TransformerBackend], batch_size: int, max_length: int
+        self,
+        backends: Sequence[TransformerBackend],
+        batch_size: int,
```

Comment:
nit:

```suggestion
        *,
        batch_size: int,
```

So nobody passes `max_length` instead of `batch_size`, etc.

Reply:
+, will do